website/global-environmental-sample-database.md

```diff
@@ -0,0 +1,131 @@
+GLOBAL ENVIRONMENTAL SAMPLE DATABASE (GESD). This database will hold
+information about the types of environmental samples that are currently
+available globally for the EMP. To make this feasible we are developing
+a submission system, which will outline the exact fields we will require
+to enable maximum comparison of sampled locations. Each sample series
+will be examined by the EMP community leaders and the final selection
+and will be made based on access to associated environmental context
+data, and experimental metadata which are compliant with the Minimal
+Information about a Metagenomic and Marker Gene Sequence (MIMS, MIMARKS
+respectively). The richer the available metadata the easier it will be
+to examine how the relationships between environment and biology.
+Specifically we want to explore environmental parameter space and it's
+role in structuring microbial ecology in ecosystems, and importantly the
+feedback role of the microbial community on the environment. This is
+only possible if we have sufficient parametrization of both the
+community and the environment; the better the parametrization the easier
+our job will be.
+
+Can we obtain information from the metadata alone?
+--------------------------------------------------
+
+Microbes exist by responding to environmental conditions; i.e if the
+temperature changes because the water in a river moves from high
+altitude to lower altitude the microbial system will respond. Microbes
+do not respond to changes in the geopolitical status of their current
+location, unless it has a direct effect on the environmental conditions.
+Therefore, the metadata collected through the GESD will enable us to
+explore the range and types of environments from which we can explore
+the Earth's microbiome. This can be used to determine what additional
+samples we will need to collect to 'fill in the gaps' in our
+exploration. For example, we may have soil samples from a pH range of
+4-6.5 and 7.3-11.2 at a range of different temperatures, nutrient loads,
+and soil types. To understand the full range of soil microbiota on earth
+we would need to explore samples whose pH is below 4, above 11.2 and
+between 6.5 and 7.3, and all at a range of different temperatures,
+nutrient loads and soil types. One important factor will be time. A
+biogeographic survey is only as useful as your understanding of the
+history of the sample you have taken, i.e. is the community you explore
+representative of that site from that time of year. Where we do not have
+extensive temporal data, we will need to explore opportunities to obtain
+samples which expand our understanding of the temporal changes in
+community structure and their relationship to environmental conditions.
+The metadata will help us to determine where this is needed.
+
+**Submitting sample information to the GESD**
+---------------------------------------------
+
+Please contact "Gail Lesley Ackermann" \<<ackermag@ucsd.edu>\> who will
```

Comment:
Gail has a guide, and I have one on the EMP500 page. We just need to decide on the last few details and then merge them.